solvers.tex

```diff
@@ -60,7 +60,7 @@
 >>> solveset(x - x, x, domain=S.Reals)
 (-oo, oo)
 >>> solveset(x - x, x, domain=S.Complexes)
-S.Complexes
```

Comment:
Here is the issue:

```
>>> repr(S.Complexes)
'S.Complexes'
>>> str(S.Complexes)
'S.Complexes'
>>> sstr(S.Complexes)
'Complexes((-oo, oo) x (-oo, oo), False)'
```

The doctester should use `repr` (that's what a Python session will use).